src/pages/Characters.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+import CharacterDetails from "../components/CharacterDetails";
+import CharacterList from "../components/CharactersList";
+import Navbar from "../components/Navbar";
+
+const Characters = () => {
+  const [selectedCharacter, setSelectedCharacter] = useState<string>("1");
```

Comment:
I don't think this is the correct value to initialize the selected character, the expected result is not rendering nothing if there is no a selected one so this must be:
const [selectedCharacter, setSelectedCharacter] = useState<string>("");

Reply:
ok, I got it, I will be more carefully following the figma design.